client/reader/search-stream/index.jsx

```diff
@@ -1,8 +1,199 @@
 /**
+ * External Dependencies
+ */
+import React, { PropTypes } from 'react';
+import { trim } from 'lodash';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+import cx from 'classnames';
+
+/**
  * Internal Dependencies
  */
-import config from 'config';
-import WithoutSites from './without-sites';
-import WithSites from './with-sites';
+import ControlItem from 'components/segmented-control/item';
+import SegmentedControl from 'components/segmented-control';
+import CompactCard from 'components/card/compact';
+import DocumentHead from 'components/data/document-head';
+import SearchInput from 'components/search';
+import { recordAction, recordTrack } from 'reader/stats';
+import SiteResults from './site-results';
+import PostResults from './post-results';
+import ReaderMain from 'components/reader-main';
+import { addQueryArgs } from 'lib/url';
+import SearchStreamHeader, { POSTS } from './search-stream-header';
```

Comment:
Discussed with @samouri - will address this in a followup PR and move into a SEARCH_TYPES perhaps.